watchtower-plugin/src/net/http.rs

```diff
@@ -376,6 +395,20 @@ mod tests {
         assert_eq!(response, add_appointment_response);
         assert_eq!(receipt, appointment_receipt);
     }
+    
+    #[tokio::test] 
+    async fn test_ping() {
+       let server = MockServer::start();
+       let api_mock = server.mock(|when, then| {
+           when.method(GET).path(Endpoint::Ping.path());
+            then.status(200);
```

Comment:
Indentation is a little bit off here. Use `cargo fmt`.

Reply:
Oh yess! I will fix indentation